.eslintrc

```diff
@@ -152,8 +152,32 @@
     {
       "files": ["packages/web-frontend/**"],
       "rules": {
+        "camelcase": "warn",
         "import/no-named-as-default": "off",
-        "react/destructuring-assignment": "off"
+        "import/no-cycle": "warn",
+        "jsx-a11y/click-events-have-key-events": "warn",
```

Comment:
By default react-scripts has these lint properties set to error which crashes the build. It would be good to fix all these lint errors at some point but that is out of scope of this PR.

Reply:
Do any of them warrant being changed in our central eslint config?

Reply:
hmmm I don't think so. The other create-react-app apps seem to have the linting options as warn by default. From my lookup, this is because they are on a different version of react-scripts to the one I just installed. It may be that in the future react will enforce these lint rules more strictly so perhaps at that time we can tackle it.